web-console/4.0-designs/storage/Expand Storage Cluster/Expand Storage Cluster.md

```diff
@@ -0,0 +1,69 @@
+# Expand OCS Cluster
+
+## Supported Deployments for OCS 4.2
+
+- AWS (IPI & UPI)
+- VMware (UPI only)
+- RHHI.Next (Bare metal)
+
+## Expand Resource Wireframes 
+We added to the storage section a new entity for “Storage Resources” which lists all the different storage platforms that are applicable or currently configured on OpenShift:
+
+## Add Capacity
+To add capacity, the flow will be decided based on the platform OpenShift is deployed on:
+
+- **Add Capacity** - add capacity by creating underlying storage devices/volumes (in the cloud, i.e. AWS/cloud)
+- **Add Storage Devices** - use existing nodes & add available storage devices/volumes to select from (for UPI/unmanaged, i.e. VMware / Baremetal)
+
+### Add Capacity (AWS/cloud)
+![Storage Resources List](img/Expand-Storage-01.png) 
+In the resource kebab menu, the user will find an action called “Add Capacity” clicking on that will open the add modal.
+
+![Add Capacity Modal](img/Expand-Storage-02.png) 
```

Comment:
@yuvalgalanti

Change "Resource 2 - AWS EBS" to the actual cluster name shown in the list view, i.e. "ocs-converged-aws" to be consistent with the triggered kebab menu previously.  You're changing the "cluster name" and not the EBS resource, so the terminology used may confuse the audience.

Shall we change "Capacity" to "Requested Capacity" to match how we have it in the install workflow 

![image](https://user-images.githubusercontent.com/5589919/61480020-073ff800-a963-11e9-93e9-3241523ed774.png)